src/condor_tests/ornithology-docs/source/api/custom_plugins.rst

```diff
@@ -0,0 +1,213 @@
+File Transfer Plugins
+=====================
+
+Ornithology comes with the following built in file transfer plugins.
+These plugins are readily available for jobs to use as part of the
+default file transfer plugin list, but can also be referred to by
```

Comment:
See subsequent comment about specifying that this is the configuration for Ornithology's default condor (unless it's also the configuration for all HTCondors started by Ornithology unless over-ridden, which would be fine but would require different phrasing).  They aren't in the param table, anyway. :)